website/src/components/BlogsSlider/index.tsx

```diff
@@ -82,10 +83,7 @@ const BlogsSlider: React.FC<BlogsSliderProps> = ({ recommendedBlogs }) => {
                     <span>
                       <ReactMarkdown
                         children={
-                          elm.content
-                            .substring(0, 200)
-                            .replace(/[\n]/g, ". ")
-                            .replace(/[^a-zA-Z ]/g, "") + "..."
+                          getContentPreview(elm.excerpt)
```

Comment:
I think we should add word limit here to maintain the same content length as per the design, even if `excerpt` length provided is lengthy